### ISSUE TITLE
appx releases on releases tag

### ISSUE THREAD

**Issue by Poopooracoocoo** [microsoft/Xaml-Controls-Gallery#110]

**Comment by chingucoding** [comment]
I agree, maybe we/you could publich the appx of the app so users can install it without having to build it if they want the new version :)

**Comment by Poopooracoocoo** [comment]
@stmoy there was a new release. any chance that we'll see a release here too?

**Comment by chingucoding** [comment]
@stmoy I agree, adding the appx to the repo does not make a lot whole of sense. However we could upload an appx to the [GitHub Releases](https://github.com/microsoft/Xaml-Controls-Gallery/releases) as a form of "Insider Preview". But this would be rather a caveat and I don't think this would be necessary.

**Comment by Poopooracoocoo** [comment]
I made a similar issue on the Calculator repo and made points there

**Comment by Poopooracoocoo** [comment]
Terminal and winget do this. Would love to see a new release as the app is pretty outdated now.

**Comment by stmoy** [comment]
This is still on my radar, but no updates yet. There's a few "infrastructure" related things that are on the list (including fixing the CI builds, building appx's, etc.) but it's been difficult to carve out the time to get it done 😅   Fortunately, the team has grown a little bit, so I'm hopeful that @kat-y will be able to help out here.

**Comment by gregwoo-microsoft** [comment]
I agree that this is a good idea but not a top priority focus right now. I'll leave this issue open to make sure we don't forget about this suggestion :)
